src/ast/codegen_llvm.cpp

```diff
@@ -681,7 +681,19 @@ void CodegenLLVM::visit(Call &call)
     }
     expr_ = stackid;
   }
-
+  else if (call.func == "override_return") {
```

Comment:
Could you move the `{` onto a separate line to match the surrounding code please? (here and below)
Nitpicky, but I think I might go through and reformat all the code one day to be consistent so I'm trying to minimise my future changes ;)

Reply:
Please do, I always mess it up. Would be good to get  #639  done :)